### ISSUE TITLE
bug: "My spaces" page not loading spaces

### ISSUE THREAD

**Issue by wa0x6e** [snapshot-labs/sx-monorepo#373]
The "My spaces" page (https://snapshotx.xyz/#/settings) is not loading, and crashing with an error from hub.snapshot.org graphql, returning

```
Variable "$where" got invalid value { controller: "" }; Field "controller" is not defined by type "RankingWhere".
```

![Screenshot 2024-05-28 at 00 35 04](https://github.com/snapshot-labs/sx-monorepo/assets/495709/50ed02fc-87d5-4d98-82dd-8be8b23b321f)

The `controller` filter is not available for hub.snapshot.org

https://github.com/snapshot-labs/sx-monorepo/blob/24c1c26b7919d19c3759fdb1e39d94998d9fc7ef/apps/ui/src/views/Settings/Spaces.vue#L8

On v1, we're currently passing directly the list of space Id, after looking for all ens domains attached to the given address, instead of asking the API for it.

**Comment by wa0x6e** [comment]
Waiting for https://github.com/snapshot-labs/snapshot-hub/pull/857

**Comment by wa0x6e** [comment]
Closed via #381
